doc/classes/AudioServer.xml

```diff
@@ -29,18 +30,21 @@
 			<argument index="2" name="at_position" type="int" default="-1">
 			</argument>
 			<description>
+				Adds an [AudioEffect] effect to the bus at [code]bus_idx[/code] at effectposition [code]at_position[/code].
```

Comment:
Repeating the words isn't necessary: "Adds an [AudioEffect] to the bus" and "at position [code]at_position[/code]."